src/Altinn.Profile.Integrations/Migration/profiledb.sql

```diff
@@ -0,0 +1,54 @@
+﻿-- Drop the database if it exists
+DROP DATABASE IF EXISTS profiledb;
```

Comment:
Remove this line and the statement for creating the database. The database is created for us with terraform.